tests/unit/test_browse.py

```diff
@@ -586,11 +586,26 @@ def test_get_color_map_from_image(self):
         # random image with values of 0 to 4.
         image_data = self.random.integers(5, size=(5, 6), dtype='uint8')
         palette_sequence = [
-            255, 0, 0, 255,
-            0, 255, 0, 255,
-            0, 0, 255, 255,
-            225, 100, 25, 25,
-            0, 0, 0, 0
+            255,
```

Comment:
I hate black here.  I wonder if it has a way to be skipped? 
yes thanks stack overflow.

```
        # fmt: off
        palette_sequence = [
            255, 0, 0, 255,
            0, 255, 0, 255,
            0, 0, 255, 255,
            225, 100, 25, 25,
            0, 0, 0, 0
        ]
        # fmt: on
```
I would like that please.

Reply:
I did it myself